.github/workflows/release.yml

```
@@ -32,6 +32,15 @@ jobs:
         uses: actions/setup-node@v3
         with:
           node-version: 18
+      - name: Publish docs api to repo
+        run: |
+          npm ci
+          npm run doc
+          git status
+          git add docs/api/md
+          git commit -am "updated api docs"
```

Two nitpicks:

1. Can we move this to the conventional commits style?
2. `-a` adds all files, which I think we'd rather not do.

```suggestion
          git commit -m "docs: update API docs"
```

---

Yeah, this became out of sync with the actual flow that uses an external action that actually commits the file (see the docs.yml flow). I'll set the message so that it uses conventional commits